Sources/HummingbirdCore/Server/HTTPServer.swift

```diff
@@ -229,9 +235,11 @@ public final class HBHTTPServer {
 }
 
 /// Protocol for bootstrap.
-protocol HTTPServerBootstrap {
+public protocol HTTPServerBootstrap {
     func bind(host: String, port: Int) -> EventLoopFuture<Channel>
```

Comment:
Given this is a public protocol it would be good that each function has a comment. You can probably copy the comments directly from swift-nio.